tests/testapp/settings.py

```diff
@@ -84,3 +84,5 @@
         'BACKEND': 'tests.management.commands.test_clear_cache.OtherCacheMock',
     },
 }
+
+TIME_ZONE = 'Asia/Seoul'
```

Comment:
Better to set the `TIME_ZONE` explicitly on the tests then to force all the tests to run on the `Asia/Seoul` timezone.

Something like: https://pytest-django.readthedocs.io/en/latest/helpers.html#settings might help

Reply:
I fixed it.